content/management/configuration-api/v3.2/index.mdx

```diff
@@ -2785,6 +2785,236 @@ curl -X POST \
 
 </Section>
 
+# Auto access
```

Comment:
I'd also move this whole section above Bot Agents, so that it's sorted alphabetically